src/main/java/com/stocks/project/controller/UserController.java

```diff
@@ -0,0 +1,48 @@
+package com.stocks.project.controller;
+
+import com.stocks.project.model.User;
+import com.stocks.project.service.UserService;
+import jakarta.servlet.http.HttpServletResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.IOException;
+import java.util.List;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
```

Comment:
Methods must return the correct status codes. Now your controller methods return 200, even if you fail to do something, it will return 200). For example, you need to return 201 for creation, 204 for deletion, etc. Look what ResponseEntity is